client/events.lua

```diff
@@ -129,26 +131,22 @@ end)
 
 -- Vehicle Commands
 
----@param vehName string
-RegisterNetEvent('QBCore:Command:SpawnVehicle', function(vehName)
-    local hash = joaat(vehName)
-    if not IsModelInCdimage(hash) then return end
-    RequestModel(hash)
-    while not HasModelLoaded(hash) do
-        Wait(0)
-    end
-
+utils.entityStateHandler('initVehicle', function(entity, _, value)
+    if not value or NetworkGetEntityOwner(entity) ~= cache.playerId then return end
```

Comment:
```suggestion
    if not value then return end
    
    for i = -1, 0 do
        local ped = GetPedInVehicleSeat(entity, i)

        if ped ~= cache.ped and ped > 0 and NetworkGetEntityOwner(ped) == cache.playerId then
            DeleteEntity(ped)
        end
    end
    
    if NetworkGetEntityOwner(entity) ~= cache.playerId then return end
```

We need to add this back because sometimes with networked server side vehicles ped spawns inside the vehicle BECAUSE some dumb cfx fuckery